openedx_external_enrollments/tests/external_enrollments/tests_mit_hz_external_enrollments.py

```diff
@@ -77,3 +81,67 @@ def test_get_user_id(self):
             self.base._get_user_id(email),  # pylint: disable=protected-access
             expected_data,
         )
+
+    @patch(
+        'openedx_external_enrollments.external_enrollments.'
+        'mit_hz_external_enrollment.MITHzInstanceExternalEnrollment._check_user_subscription'
+    )
+    def test_get_course_home_url(self, get_course_home_url_mock):
```

Comment:
I've updated the mock, it was `check_user_subscription_mock`, in this case, to assert the return value. Since this call does not affect the return value we just mock it.